src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { Component } from 'react';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { withStyles } from '@material-ui/core/styles';
+import { arrayOf, object } from 'prop-types';
+import { camelCase } from 'change-case';
+import { memoizeWith, pipe, sort as rSort, map } from 'ramda';
+import DataTable from '../DataTable';
+import sort from '../../utils/sort';
+
+const sorted = pipe(
+  rSort((a, b) => sort(a.summary, b.summary)),
+  map(({ project, summary }) => `${summary}-${project}`)
+);
+
+@withStyles(() => ({
+  summary: {
+    whiteSpace: 'nowrap',
+  },
+}))
+export default class BugsTable extends Component {
+  static propTypes = {
+    /**
+     * A list of objects to display. Each element in the list is represented
+     * by a row and each element's key-value pair represents a column.
+     */
+    items: arrayOf(object).isRequired,
+  };
+
+  state = {
+    sortBy: null,
+    sortDirection: null,
+  };
+
+  getTableData = memoizeWith(
+    (sortBy, sortDirection, items) => {
+      const ids = sorted(items);
+
+      return `${ids.join('-')}-${sortBy}-${sortDirection}`;
+    },
+    (sortBy, sortDirection, items) => {
+      const sortByProperty = camelCase(sortBy);
+
+      if (!items) {
+        return null;
+      }
+
+      if (!sortBy) {
+        return items;
+      }
+
+      return items.sort((a, b) => {
+        const firstElement =
+          sortDirection === 'desc' ? b[sortByProperty] : a[sortByProperty];
+        const secondElement =
+          sortDirection === 'desc' ? a[sortByProperty] : b[sortByProperty];
+
+        return sort(firstElement, secondElement);
+      });
+    }
+  );
+
+  handleHeaderClick = sortBy => {
+    const toggled = this.state.sortDirection === 'desc' ? 'asc' : 'desc';
+    const sortDirection = this.state.sortBy === sortBy ? toggled : 'desc';
+
+    this.setState({ sortBy, sortDirection });
+  };
+
+  render() {
+    const { items, classes } = this.props;
+    const { sortBy, sortDirection } = this.state;
+    const data = this.getTableData(sortBy, sortDirection, items);
+
+    return (
+      <DataTable
```

Comment:
Resizing the page doesn't seem to make the table smaller:

![screen shot 2018-06-04 at 10 04 06 am](https://user-images.githubusercontent.com/3766511/40921872-ae54c02a-67de-11e8-8f27-474c7e2b1952.png)

One solution is to do:

```jsx
<div className={classes.tableWrapper}>
  <DataTable ... />
</div>
```

```js
tableWrapper: {
  overflowX: 'auto',
}
```